src/app/zap-templates/zcl/data-model/chip/onoff-cluster.xml

```diff
@@ -52,7 +59,7 @@ limitations under the License.
     <attribute side="server" code="0x4000" define="GLOBAL_SCENE_CONTROL" type="boolean" default="1"                 optional="true">GlobalSceneControl</attribute>
     <attribute side="server" code="0x4001" define="ON_TIME"              type="int16u"  default="0" writable="true" optional="true">OnTime</attribute>
     <attribute side="server" code="0x4002" define="OFF_WAIT_TIME"        type="int16u"  default="0" writable="true" optional="true">OffWaitTime</attribute>
-    <attribute side="server" code="0x4003" define="START_UP_ON_OFF"      type="enum8"   default="0" writable="true" isNullable="true" optional="true">StartUpOnOff</attribute>
+    <attribute side="server" code="0x4003" define="START_UP_ON_OFF"      type="OnOffStartUpOnOff" min="0" max="2" isNullable="true" writable="true" optional="true">StartUpOnOff</attribute>
```

Comment:
we can do it this way ? great !

Reply:
Indeed, this helps establish a relationship between the attribute and its type definitions. As we review more cluster definition files, it is likely that there are other clusters that might need similar updates.

Reply:
Indeed i m gonna prepare a PR for WindowCovering following this